packages/netlify-cms-core/src/reducers/collections.ts

```diff
@@ -122,10 +122,24 @@ export const selectAllowDeletion = (collection: Collection) =>
   selectors[collection.get('type')].allowDeletion(collection);
 export const selectTemplateName = (collection: Collection, slug: string) =>
   selectors[collection.get('type')].templateName(collection, slug);
+
+export const getFieldsNames = (fields: EntryField[], prefix = '') => {
+  let names = fields.map(f => `${prefix}${f.get('name')}`);
+
+  fields.forEach((f, index) => {
+    if (f.has('fields')) {
```

Comment:
Also need to account for `field` here.

Reply:
Actually thought about adding that, but couldn't find anything in the docs about specifying a single field (only in the CMS code).
Wasn't sure if this was still supported.

Reply:
It’s documented under the list widget, although I doubt many have discovered it.